pkgs/tools/security/biscuit-cli/default.nix

```diff
@@ -1,25 +1,40 @@
-{ lib, fetchFromGitHub, rustPlatform }:
+{ lib
+, fetchFromGitHub
+, rustPlatform
+, testers
+, nix-update-script
+, biscuit-cli
+}:
 
 rustPlatform.buildRustPackage rec {
   pname = "biscuit-cli";
-  version = "0.2.0-next-pre20230103";
+  version = "0.4.0";
 
   src = fetchFromGitHub {
     owner = "biscuit-auth";
     repo = "biscuit-cli";
-    rev = "0ecf1ec4c98a90b1bf3614558a029b47c57288df";
-    sha256 = "sha256-ADJWqx70IwuvCBeK9rb9WBIsD+oQROQSduSQ8Bu8mfk=";
+    rev = version;
+    sha256 = "sha256-Fd5wBvQe7S/UZ42FMlU+f9qTwcLIMnQrCWVRoHxOx64=";
   };
 
-  cargoLock = {
-    outputHashes."biscuit-auth-3.0.0-alpha4" = "sha256-4SzOupoD33D0KHZyVLriGzUHy9XXnWK1pbgqOjJH4PI=";
-    lockFile = ./Cargo.lock;
+  cargoHash = "sha256-SHRqdKRAHkWK/pEVFYo3d+r761K4j9BkTg2angQOubk=";
+
+  patches = [ ./version-0.4.0.patch ];
```

Comment:
Can you add a comment linking to https://github.com/biscuit-auth/biscuit-cli/issues/44?

Reply:
Sure!